apache-ansible-demo/docs/scripts.md

```diff
@@ -0,0 +1,13 @@
+The set of the scripts in this folder allows to invoke the resource package's playbooks from command line.
```

Comment:
Should there be a link here to ```../Contains/hw-apache-vnfc/Lifecycle/scripts``` folder? On current reading you would expect the scripts to be under the docs folder.